topologic/embedding/laplacian_spectral_embedding.py

```diff
@@ -1,15 +1,16 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
 
+from typing import Optional
```

Comment:
Is this actually used?

Reply:
Yep, it is used for type hints for the spectral methods for elbow_cut and svd_seed.

Reply:
Ugh nevermind, I see it just moved.  I thought you added it now.